.github/workflows/release.yml

```diff
@@ -40,19 +51,22 @@ jobs:
 
     - name: Switch to linking with wild
       run: |
-        mkdir .cargo
-        echo -e "[target.${{matrix.target}}]\nlinker = \"clang\"\nrustflags = [\"-Clink-arg=--ld-path=${PWD}/target/release/wild\"]\n" >>"$HOME/.cargo/config.toml"
+        mkdir -p "$HOME/.cargo"
+        WILD="${PWD}/target/release/wild"
+        $WILD --version
+        echo -e "[target.${{matrix.target}}]\nlinker = \"clang\"\nrustflags = [\"-Clink-arg=--ld-path=$WILD\"]\n" >>"$HOME/.cargo/config.toml"
 
     - name: Build release binary
-      run: cargo build --profile dist --target ${{ matrix.target }}
+      run: cargo ${{matrix.build || 'build' }} ${{matrix.args}} --profile dist --target ${{ matrix.target }}
 
     - name: Create tarballs
       run: |
         n="wild-linker-${{ github.ref_name }}-${{ matrix.target }}"
         mkdir "$n"
         cp "target/${{ matrix.target }}/dist/wild" "$n"
         cp README.md LICENSE* CHANGELOG.md "$n"
-        strip "$n/wild"
+        strings "$n/wild" | grep "Linker: Wild"
+        strip "$n/wild" || riscv64-linux-gnu-strip "$n/wild" || echo "::warning::couldn't strip binary"
```

Comment:
One alternative that might be simpler would be to add `strip = true` to `[profile.dist]` is `Cargo.toml`. Then the linker can just not write the symbol table.